src/constants.ts

```diff
@@ -39,6 +39,17 @@ export const SITE_ENV_VAR = 'DD_SITE'
 export const LOGS_INJECTION_ENV_VAR = 'DD_LOGS_INJECTION'
 export const LOGS_PATH_ENV_VAR = 'DD_SERVERLESS_LOG_PATH'
 export const HEALTH_PORT_ENV_VAR = 'DD_HEALTH_PORT'
+export const LOG_LEVEL_ENV_VAR = 'DD_LOG_LEVEL'
+export const TRACE_ENABLED_ENV_VAR = 'DD_TRACE_ENABLED'
+export const DD_LLMOBS_ENABLED_ENV_VAR = 'DD_LLMOBS_ENABLED'
+export const DD_LLMOBS_ML_APP_ENV_VAR = 'DD_LLMOBS_ML_APP'
+export const DD_LLMOBS_AGENTLESS_ENABLED_ENV_VAR = 'DD_LLMOBS_AGENTLESS_ENABLED'
+export const EXTRA_TAGS_ENV_VAR = 'DD_TAGS'
```

Comment:
Can we rename this to `DD_TAGS_ENV_VAR`? At Datadog, idk why, there's the concept of `DD_EXTRA_TAGS` which is also used and at the end it merges back to `DD_TAGS`, but just wanna keep consistency here